diff --git a/src/helpers/SdDaemon.cpp b/src/helpers/SdDaemon.cpp
index 48c23e6b..e3cc2bc7 100644
--- a/src/helpers/SdDaemon.cpp
+++ b/src/helpers/SdDaemon.cpp
@@ -1,5 +1,6 @@
 #include "SdDaemon.hpp"
 
+#include <cstdlib>
 #include <memory>
 
 #include <fcntl.h>
@@ -35,7 +36,7 @@ namespace Systemd {
         };
         std::unique_ptr<int, decltype(cleanup)> fdCleaup(&fd, cleanup);
 
-        const char*                             addr = getenv(envVar);
+        const char*                             addr = std::getenv(envVar);
         if (!addr)
             return 0;
 
